ci/pipeline.yml

```diff
@@ -401,24 +402,6 @@ jobs:
       - terraform-secrets/terraform.yml
       - terraform-yaml/state.yml
       - common-tooling/tooling-bosh.yml
-      ops_files:
```

Comment:
Why are you deleting this chunk?

Reply:
the `bosh-deployment` anchor has this stuff